viite-UI/src/view/ProjectChangeTable.js

```diff
@@ -249,7 +267,7 @@
         '<td class="project-change-table-dimension">' + changeInfoSeq.target.endAddressM + '</td>' +
         '<td class="project-change-table-dimension">' + (changeInfoSeq.target.endAddressM - changeInfoSeq.target.startAddressM) + '</td>' +
         '<td class="project-change-table-dimension">' + replaceParallelLink(changeInfoSeq.target.discontinuity) + '</td>' +
-        '<td class="project-change-table-dimension">' + changeInfoSeq.target.roadType + '</td>' +
+        '<td class="project-change-table-dimension">' + getAdministrativeClassText(changeInfoSeq.target.roadType) + '</td>' +
```

Comment:
ja vastaavasti tähän joku "TODO: VIITE-2568, getAdministrativeClassText call to be changed to changeInfoSeq.target.administrativeClass, when backend side of AdministrativeClass addition completed." ?